test/unit/PackagePluginSpec.hs

```diff
@@ -63,27 +63,48 @@ packageSpec = do
             uri       = filePathToUri $ fp </> "add-package-test.cabal"
             args      = AddParams fp (fp </> "AddPackage.hs") "text"
             act       = addCmd args
-            textEdits =
-              List
-                [ TextEdit (Range (Position 0 0) (Position 7 27)) $ T.concat
-                  [ "cabal-version: >=1.10\n"
-                  , "name: add-package-test\n"
-                  , "version: 0.1.0.0\n"
-                  , "license: BSD3\n"
-                  , "maintainer: luke_lau@icloud.com\n"
-                  , "author: Luke Lau\n"
-                  , "build-type: Simple\n"
-                  , "extra-source-files:\n"
-                  , "    ChangeLog.md"
+            textEdits = case ghcVersion of
+              GHC88 ->
+                List
+                  [ TextEdit (Range (Position 0 0) (Position 7 27)) $ T.concat
+                    [ "cabal-version:      >=1.10\n"
+                    , "name:               add-package-test\n"
+                    , "version:            0.1.0.0\n"
+                    , "license:            BSD3\n"
+                    , "maintainer:         luke_lau@icloud.com\n"
+                    , "author:             Luke Lau\n"
+                    , "build-type:         Simple\n"
+                    , "extra-source-files: ChangeLog.md"
+                    ]
+                  , TextEdit (Range (Position 10 0) (Position 13 34)) $ T.concat
+                    [ "    main-is:          AddPackage.hs\n"
+                    , "    default-language: Haskell2010\n"
+                    , "    build-depends:\n"
+                    , "        base >=4.7 && <5,\n"
+                    , "        text : {} -any"
```

Comment:
what the hell is this text line? `text : {} -any`? Is that some new syntax?

Reply:
I'm not clear, since I don't use cabal

Reply:
It is fine, it was more out of interest.